package.json

```diff
@@ -12,7 +12,7 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC",
+  "license": "MIT",
```

Comment:
update license according to https://github.com/Schneegans/dynamic-badges-action/blob/master/LICENSE

Reply:
Thanks for this fix!